cafebabel/migrations.py

```diff
@@ -0,0 +1,42 @@
+import json
+from datetime import datetime, timezone
+
+import click
+from cafebabel.users.models import Role
+from progressist import ProgressBar
+
+
+def iterate_over_json_file(filename):
+    with open(filename) as json_file:
+        return json.loads(json_file.read())
+
+
+def migrate_users(app, limit):
+    click.echo('Starting users migration.')
+    ds = app.user_datastore
+    old_users = iterate_over_json_file('users.json')
```

Comment:
I assume this file name/path should be the parameter from the command line instead of hardcoded even if it's hopefully a one time script.
That would also fix the gitignore

Reply:
Alright, I was hesitant between convention vs. configuration for a one-timer